packages/core/src/background/handlers/State.ts

```diff
@@ -85,14 +88,14 @@ export default class State {
       .map(({ id, request, url }): ProvideUidRequest => ({ id, request, url }));
   }
 
-  private popupClose (): void {
+  private _popupClose (): void {
```

Comment:
just curious, why this rename? doesn't seem there's a type issue here like the `_handle` case, is there?

Reply:
@hlminh2000 it's because those methods exist in the parent class with private visibility. They can neither be accessed or overriden with the same name 🤷 .